src/ng/directive/input.js

```diff
@@ -1476,6 +1476,8 @@ function radioInputType(scope, element, attr, ctrl) {
 
   ctrl.$render = function() {
     var value = attr.value;
+    // Strict comparison would cause a BC
```

Comment:
Are you sure ? From a (super) quick look, we seems to be updating `$viewValue` with `attr.value` and here we are comparing it with `attr.value` again.